### DISCUSSION TITLE
A couple of comments

### DISCUSSION THREAD

**Discussion by lkzkk** [ldodds/osm-queries#4] [Ideas]
Hi - putting some comments here as discussed on your OSM diary:

- I find lesson 01 a little too complicated to begin with. I could follow it because I have used Overpass Turbo before (mostly with their excellent wizard, but still), but I think someone new to it might be overwhelmed. Is there a way to start with even more minimal example?
- I'm a little confused how 01 reads "The second way to define a bounding box is as a parameter to a statement." and the parameter comes in parenthesis, but 02 reads "The out statement can be controlled by adding some parameters to it. These are appended to the statement." and the parameters use no parenthesis. Why do parameters sometimes are marked with, and others without parenthesis?
- On 03: unless I'm mistaken, the out statement doesn't always need a parameter. Would you be able to add information on what is the default output if there is no parameter added?

Thanks for your consideration!

**Comment by ldodds** [comment]
Thanks @lukaszkruk this is very helpful feedback. 

I'll look at revising parts 1, 2 and 3 to clarify about parameters, default output, etc.

I'm not sure how best to simplify lesson 1. My goal has been that every step is built around an executable query and its hard to get much smaller than the current example. There needs to be at least two statements, `node` and `out` but I can simplify the `out` statement to remove `skel` parameter.

If I remove the `bbox` query setting, then I would need to add a parameter to the `node` statement, so that it returns only nodes in the tutorial area. 

Would it help if it were broken up so that there was:

- a simple example, just nodes in a bounding box to introduce basic syntax
- follow-on example to illustrate how to set bounding box in different ways?
- and then put the TRYME into other examples

What do you think?

**Comment by lkzkk** [comment]
Lesson 00 introduces node parameters. Perhaps a logical next step would be to build on this, and show that you can query for a node not only by ID, but also by bounding box, or by name? These could be 2-3 lessons that gradually expand just what parameters to the node query itself do, and if i'm not mistaken should be possible to do without touching on the settings and out statement parameters?

Then the out statement and settings could be introduced later, each in its own lesson or two. As it is, lesson 01 introduces all of them at once.

Hope this explains my point a little clearer? Please let me know if it doesn't, I'll try to rephrase if needed.

**Reply by ldodds** [reply]
Thanks @lukaszkruk that's helpful. I'll aim to move the first few lessons in that direction. I'll do that this week and publish a new version.

While not strictly necessary to include the `out` statement in the early examples, without it there's just no results at all, so I think its best to include that. But this can be done in a straight-forward way I think
